pkgs/tools/admin/lxd/default.nix

```diff
@@ -1,21 +1,21 @@
 { stdenv, pkgconfig, lxc, buildGoPackage, fetchurl
 , makeWrapper, acl, rsync, gnutar, xz, btrfs-progs, gzip, dnsmasq
 , squashfsTools, iproute, iptables, ebtables, libcap, libco-canonical, dqlite
-, raft-canonical, sqlite-replication
+, raft-canonical, sqlite-replication, udev
 , writeShellScriptBin, apparmor-profiles, apparmor-parser
 , criu
 , bash
 }:
 
 buildGoPackage rec {
   pname = "lxd";
-  version = "3.18";
+  version = "3.21";
```

Comment:
3.22 was recently released, would you mind updating it? https://discuss.linuxcontainers.org/t/lxd-3-22-has-been-released/7027

Reply:
Sure, wouldn't be the first time on this PR...

Reply:
Done!